docusaurus/docs/create-a-plugin/extend-a-plugin/include-dashboards.md

```diff
@@ -0,0 +1,132 @@
+---
+id: include-dashboards
+title: Including dashboards
+description: Include dashboards in Grafana datasource and app plugins
+keywords:
+  - grafana
+  - plugin
+  - dashboards
+  - app
+  - datasource
+---
+
+# Including dashboards with a plugin
+
+This guide focuses on integrating dashboards into app and datasource plugins. By bundling pre-configured dashboards, users can more easily understand the full potential of a plugin without the need to build dashboards from scratch. We'll walk you through the process of including these dashboards in your plugins.
+
+## Create a dashboard
+
+Start by creating the dashboard you wish to bundle with your plugin. The development environment provided by create-plugin can aid with creating and testings dashboards. Below we cover the steps for both datasource and app plugins, including tips for crafting an effective dashboard.
+
+### Datasource plugins
+
+#### Export the dashboard
+
+In this step, we export the dashboard to a JSON file so it can be placed in a file along your plugin source code.
+
+1. Open your dashboard within Grafana.
+1. Click the Share icon in at the top-left of the dashboard.
+1. Click **Export**.
+1. Enable **Export for sharing externally**, and then click **Save to file**.
+
+Exporting with this option replaces direct data source references with placeholders. This ensures the dashboard can make use of the user's data source instance when imported.
+
+### App plugins
+
+#### Setting Up a Datasource Variable
+
+To facilitate user customization, create a dashboard data source variable. This allows users to link their own data source instances easily once imported.
+
+1. Create a datasource variable by choosing the data source variable type and give it a name
+   ![Datasource variable](/img/app-dashboard-ds-variable.png)
+1. Each panel that you create should use the datasource variable as its datasource
+   ![Datasource selection](/img/app-dashboard-ds-select.png)
+
+#### Export the dashboard
+
+1. Navigate to your dashboard in the Grafana application
+1. Click the Share icon in at the top-left of the dashboard.
+1. Click **Export** and then click **Save to file**.
+1. Open the dashboard json file in your code editor and set it's `id` property to `null`.
```

Comment:
Would this be necessary for the dashboards for datasources as well?

Reply:
I don't think so. I'm pretty sure the `export for sharing` step in the datasource instructions handles this.